tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -33,8 +33,9 @@ else
   #Install ONNX
   #5af210ca8a1c73aa6bae8754c9346ec54d0a756e is v1.2.3
   #bae6333e149a59a3faa9c4d9c44974373dcf5256 is v1.3.0
-  #0c8d857bb162431912b255d5c0e773fb7c131a65 is v1.3.0 latest
-  for onnx_version in "5af210ca8a1c73aa6bae8754c9346ec54d0a756e" "bae6333e149a59a3faa9c4d9c44974373dcf5256" "0c8d857bb162431912b255d5c0e773fb7c131a65"; do
+  #0c8d857bb162431912b255d5c0e773fb7c131a65 is v1.3.0
```

Comment:
This line should be removed.

Reply:
Did I not just added it?

Reply:
Did I not just add it?

---
In reply to: [243453465](https://github.com/Microsoft/onnxruntime/pull/235#discussion_r243453465) [](ancestors = 243453465)

Reply:
The correct one should be
```bash
#5af210ca8a1c73aa6bae8754c9346ec54d0a756e is v1.2.3
#bae6333e149a59a3faa9c4d9c44974373dcf5256 is v1.3.0
#0a7cc483eb0c34e15414437bbbb420f52df4d8c2 is v1.3.0 latest
```
Only 3 lines of comment, not 4. In your branch, there are 4 lines.

0c8d857bb162431912b255d5c0e773fb7c131a65 is not v1.3.0.